frontend/packages/pipelines-plugin/integration-tests/features/e2e/pipeline-ci.feature

```diff
@@ -0,0 +1,136 @@
+@pipelines
+Feature: Entire pipeline flow from Builder page
+              As a user, I want to perform operations on pipeline like Creation, Execution and Deletion
+
+
+        @pre-condition
+        Scenario: Background Steps
+            Given user has created or selected namespace "pipeline-flow"
+
+
+        @smoke
+        Scenario: Create a pipeline from pipeline builder page
+            Given user is at Pipeline Builder page
+             When user enters pipeline name as "flow"
+              And user clicks Add task button under Tasks section
+              And user searches "git-clone" in quick search bar
+              And user selects "git-clone" from git community
+              And user clicks on Install and add button
+              And user clicks the "git-clone" node
+              And user enters the url as "https://github.com/sclorg/golang-ex.git" under Parameters section in cluster task sidebar
+              And user enters revision as "master" under Parameters section in cluster task sidebar
+              And user adds the parameter details like Name, Description and Default Value
+              And user clicks on Add workspace
+              And user adds the Workspace name as "git"
+              And user clicks the "git-clone" node
+              And user selects the "git" workspace in the Output of Workspaces in cluster task sidebar
+              And user clicks Create button on Pipeline Builder page
+             Then user will be redirected to Pipeline Details page with header name "flow"
+              And user will see workspace mentioned as "git" in the Workspaces section of Pipeline Details page
+
+
+        Scenario: Pipelines page details
```

Comment:
Please tag all the scenarios so they would either run (`@smoke`/`@regression`), or as  `@to-do`

Reply:
This CI feature related scenarios are all automated, but to make sure for pr merge, tagged single scenario as smoke. To follow the process, will update the tag for remaining scenarios as @to-do